main.py

```diff
@@ -115,6 +115,15 @@ def get_redirect_route(regex_route, handler, defaults=None):
 
     Warning: this method strips off parameters after the trailing slash. URLs
     with parameters should be formulated without the trailing slash.
+
+    Args:
+        regex_route: unicode. A raw string representing a route.
+        handler: BaseHandler. A callable to handle the route.
+        defaults: dict. Default, keywords or optional NoneType to be passed
```

Comment:
The description for defaults is still confusing. Perhaps try this: 
`Optional defaults parameter to be passed into the RedirectRoute object.`

Reply:
Made the change. Using those words Thx for the suggestion!

Reply:
DONE